jbmc/src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -1548,6 +1548,21 @@ void java_string_library_preprocesst::initialize_conversion_table()
   cprover_equivalent_to_java_string_returning_function
     ["java::org.cprover.CProverString.substring:(Ljava/Lang/"
      "StringBuffer;II)Ljava/lang/String;"] = ID_cprover_string_substring_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(Z)Ljava/lang/String;"] =
+      ID_cprover_string_of_bool_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(C)Ljava/lang/String;"] =
+      ID_cprover_string_of_char_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(I)Ljava/lang/String;"] =
+      ID_cprover_string_of_int_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(J)Ljava/lang/String;"] =
+      ID_cprover_string_of_long_func;
+  cprover_equivalent_to_java_string_returning_function
+    ["java::org.cprover.CProverString.toString:(F)Ljava/lang/String;"] =
+      ID_cprover_string_of_float_func;
```

Comment:
why is there no string_of_double entry?

Reply:
Double to string is handled by a model via `CProverString.toString(CProver.doubleToFloat(d))` as double to string conversion is not implemented AFAIK.